assets/js/main.js

```diff
@@ -387,26 +387,29 @@ $(function(){
 });
 // open accordion for direct links to panel id
 $(window).load(function() {
-  var getHash = location.hash; //get hash from js object location
+  const getHash = location.hash; //get hash from js object location
+
   function activateAccordion(id) {
-    if (id.length)//check if hash isn't empty
-    {
-      var accordion = $('a[href="' + id + '"]');
-      $panelHeight = accordion.parents('.panel').height();
-      console.log('Offset Top: ' + accordion.parents('.panel').offset().top);
-      $('html,body').animate({
-          scrollTop: accordion.parents('.panel').offset().top //scroll to accordion
-      }, 100, function () {
-          accordion.click(); //simulate click
-          accordion.parents('.panel').find('.panel-title a.no-scroll i.fa').addClass('panel-isOpen'); //point caret down for oepned panel
-      });
-    }
-    else{
-      //if no hash is called, open first panel
-      $('.panel-group .panel:first-child .panel-collapse').addClass('in');
-      $('.panel-group .panel:first-child .panel-title a.no-scroll i.fa').addClass('panel-isOpen');
-    }
-  };
+    const accordion = $('a[href="' + id + '"]');
+
+    //Check panel class exists 
+    if(accordion.length > 0 && accordion.parents('.panel').length > 0){
+      if (id.length)//check if hash isn't empty
+      {
+        $('html,body').animate({
+            scrollTop: accordion.parents('.panel').offset().top //scroll to accordion
+        }, 100, function () {
+            accordion.click(); //simulate click
+            accordion.parents('.panel').find('.panel-title a.no-scroll i.fa').addClass('panel-isOpen'); //point caret down for oepned panel
+        });
+      }
+      else{
```

Comment:
More formatting, should be `} else {`...

Reply:
Ahhhhh I will get IntelliJ now. My code editor is really messing me up